for_production.sh

```diff
@@ -16,96 +17,33 @@
 # - scans are in process subdirectory 'images'
 # - text results should reside in subdir 'ocr/alto'
 # To be called (after copying data to 3.) via manager, e.g.:
-#     ssh -Tn -p 9022 ocrd@ocrd-manager for_production.sh 501543 3 /home/goobi/work/daten/501543 deu Fraktur ocr.sh
+#     ssh -Tn -p 9022 ocrd@ocrd-manager for_production.sh 501543 3 /home/goobi/work/daten/501543 deu Fraktur
 
 set -eu
 set -o pipefail
 
-TASK=$(basename $0)
-PROC_ID=$1
-TASK_ID=$2
-PROCDIR="$3"
-LANGUAGE="$4"
-SCRIPT="$5"
-WORKFLOW="${6:-ocr.sh}"
+source ocr.sh
 
-logger -p user.notice -t $TASK "running with $* CONTROLLER=$CONTROLLER"
-cd /data
-
-if ! test -d "$PROCDIR"; then
-    logger -p user.error -t $TASK "invalid process directory '$PROCDIR'"
-    exit 2
-fi
-WORKFLOWFILE="$PROCDIR/ocr-workflow.sh"
-if [ -f "$WORKFLOWFILE" ]; then
-  WORKFLOW=$(realpath "$WORKFLOWFILE")
-else
-  WORKFLOW=$(command -v "$WORKFLOW" || realpath "$WORKFLOW")
-	if ! test -f "$WORKFLOW"; then
-		logger -p user.error -t $TASK "invalid workflow '$WORKFLOW'"
-		exit 3
-	fi
-fi
-if test -z "$CONTROLLER" -o "$CONTROLLER" = "${CONTROLLER#*:}"; then
-    logger -p user.error -t $TASK "envvar CONTROLLER='$CONTROLLER' must contain host:port"
-    exit 4
-fi
-CONTROLLERHOST=${CONTROLLER%:*}
-CONTROLLERPORT=${CONTROLLER#*:}
-
-# copy the data from the process directory controlled by production
-# to the transient directory controlled by the manager
-# (currently the same share, but will be distinct volumes;
-#  so the admin can decide to either mount distinct shares,
-#  which means the images will have to be physically copied,
-#  or the same share twice, which means zero-cost copying).
-WORKDIR=ocr-d/"$PROCDIR" # will use other mount-point than /data soon
-mkdir -p $(dirname "$WORKDIR")
-cp -vr --reflink=auto "$PROCDIR/images" "$WORKDIR" | logger -p user.info -t $TASK
+init $@
```

Comment:
note: `$@` must be quoted, otherwise it will not pass the params unchanged (i.e. the exact tokenization can change).

```suggestion
init "$@"
```